src/test/java/pl/touk/sputnik/connector/local/LocalFacadeTest.java

```diff
@@ -0,0 +1,118 @@
+package pl.touk.sputnik.connector.local;
+
+import com.google.common.collect.ImmutableList;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.diff.DiffEntry;
+import org.eclipse.jgit.diff.DiffEntry.ChangeType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.touk.sputnik.review.Comment;
+import pl.touk.sputnik.review.Review;
+import pl.touk.sputnik.review.ReviewFile;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Answers.RETURNS_DEEP_STUBS;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class LocalFacadeTest {
+    @Mock(answer = RETURNS_DEEP_STUBS)
+    private Git git;
+    @Mock
+    private DiffEntry modifiedFile;
+    @Mock
+    private DiffEntry deletedFile;
+    @Mock
+    private DiffEntry newFile;
+    @Mock
+    private Review review;
+
+    private LocalFacade localFacade;
+
+    @BeforeEach
+    void setUp() {
+        localFacade = spy(new LocalFacade(git));
+    }
+
+    private void setUpDiff() throws GitAPIException {
+        when(modifiedFile.getNewPath()).thenReturn("/path/to/modifiedFile");
+        when(modifiedFile.getChangeType()).thenReturn(ChangeType.MODIFY);
+
+        when(newFile.getNewPath()).thenReturn("/path/to/newFile");
+        when(newFile.getChangeType()).thenReturn(ChangeType.ADD);
+
+        when(deletedFile.getChangeType()).thenReturn(ChangeType.DELETE);
+
+        when(git.diff().call()).thenReturn(ImmutableList.of(modifiedFile, deletedFile, newFile));
+    }
+
+    @Test
+    void shouldParseListFilesResponse() throws GitAPIException {
+        setUpDiff();
+
+        List<ReviewFile> reviewFiles = localFacade.listFiles();
+        assertThat(reviewFiles).isNotEmpty();
+    }
+
+    @Test
+    void shouldNotListDeletedFiles() throws GitAPIException {
+        setUpDiff();
+
+        List<ReviewFile> reviewFiles = localFacade.listFiles();
+        assertThat(reviewFiles)
+                .hasSize(2)
+                .extracting(ReviewFile::getReviewFilename).containsExactly(modifiedFile.getNewPath(), newFile.getNewPath());
+    }
+
+    @Test
+    void publishLogsNoCommentsIfAllFilesHaveNoComments() {
+        ReviewFile review1 = new ReviewFile("/path/to/file1");
+        ReviewFile review2 = new ReviewFile("path/to/file2");
```

Comment:
I'd consider this a datamodel class, so I'd think it makes more sense to not mock it, but I don't mind changing it in this case, since we have to mutate it to add the comments. Done.